interp/impargs.ml

```diff
@@ -22,6 +22,11 @@ open EConstr
 open Reductionops
 open Constrexpr
 
+let whd_all env sigma t = t
```

Comment:
Can't you use `Reductionops.clos_whd_flags`?

Reply:
Thanks